yt_dlp/extractor/niconicochannelplus.py

```diff
@@ -54,6 +61,7 @@ class NiconicoChannelPlusIE(NiconicoChannelPlusBaseIE):
             'channel': '前田佳織里の世界攻略計画',
             'channel_id': 'kaorin',
             'channel_url': 'https://nicochannel.jp/kaorin',
+            'age_limit': None,
```

Comment:
```suggestion
```
None values is not needed in test defenition

Reply:
> None values is not needed in test defenition

OK I will remove those None values.